docs/quick_tutorial/authorization.rst

```diff
@@ -74,6 +74,17 @@ Steps
 #. Visit http://localhost:6543/howdy in a browser. You should be asked to
    login.
 
+#. Lets test if the ``editor`` can access howdy and the the ``viewer`` can't:
```

Comment:
Two grammar corrections:

- Let's
- remove extra "the"

Reply:
@tomascorrea would also please sign CONTRIBUTORS.txt, if you have not done so? Thank you! This is good work!